beluga/include/beluga/motion/differential_drive_model.hpp

```diff
@@ -86,8 +87,11 @@ class DifferentialDriveModel {
   /// 2D or flattened 3D pose as motion model state (to match that of the particles).
   using state_type = StateType;
 
+  /// Time point type for motion model control actions.
+  using timestamped_state_type = TimeStamped<state_type>;
+
   /// Current and previous odometry estimates as motion model control action.
-  using control_type = std::tuple<state_type, state_type>;
+  using control_type = std::tuple<timestamped_state_type, timestamped_state_type>;
```

Comment:
@fbattocchia earlier comment still applies. This model doesn't need timestamped state.